[Trivial] Improve logs in CoinJoinClient & AliceClient

yahiheb opened the pull request:
Commit 1: Logs remaining time for signing state to end.
Commit 2: Beautifies and makes the logs consistent.
Commit 3 & 5: Removes the word `successfully` from some logs.
Commit 4: Removes redundant log. The following log already exists: https://github.com/zkSNACKs/WalletWasabi/blob/accb284a8365c62d66f685bb446a6262793ed94e/WalletWasabi/WabiSabi/Client/AliceClient.cs#L69

MaxHillebrand commented on WalletWasabi/WabiSabi/Client/CoinJoinClient.cs:
```diff
@@ -157,7 +157,7 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 			if (!registeredAliceClientAndCircuits.Any())
 			{
-				Logger.LogInfo($"Round ({roundId}): There is no available alices to participate with.");
+				Logger.LogInfo($"Round ({roundId}): There are no available alices to participate with.");
```
```suggestion
				Logger.LogInfo($"Round ({roundId}): There are no available Alices to participate with.");
```

adamPetho replied:
+1, but we have a `LogCoinJoinSummary` method, maybe we could log the TxID there.

MaxHillebrand commented on WalletWasabi/WabiSabi/Client/CoinJoinClient.cs:
```diff
@@ -170,7 +170,8 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 			var finalRoundState = await RoundStatusUpdater.CreateRoundAwaiter(s => s.Id == roundId && s.Phase == Phase.Ended, cancellationToken).ConfigureAwait(false);
-			Logger.LogDebug($"Round ({roundId}): Round Ended - WasTransactionBroadcast: '{finalRoundState.WasTransactionBroadcast}'.");
+
+			Logger.LogDebug($"Round ({roundId}): Round ended - WasTransactionBroadcast: '{finalRoundState.WasTransactionBroadcast}'.");
```
maybe here add txid.

adamPetho replied:
+1, but we have a LogCoinJoinSummary method, maybe we could log the TxID there.

yahiheb replied:
I opened #7628

Szpoti commented on WalletWasabi/WabiSabi/Client/CoinJoinClient.cs:
```diff
@@ -225,9 +226,9 @@ async Task<(AliceClient? AliceClient, PersonCircuit? PersonCircuit)> RegisterInp
+		var remainingTimeForRegistration = roundState.InputRegistrationEnd - DateTimeOffset.UtcNow;
 
-		Logger.LogDebug($"Round ({roundState.Id}): Input registration started, it will end in {remainingTimeForRegistration:hh\\:mm\\:ss}.");
+		Logger.LogDebug($"Round ({roundState.Id}): Input registration started - it will end in: {remainingTimeForRegistration:hh\\:mm\\:ss}.");
```
I was thinking maybe of an improvement - to see (besides the remaining time?) the exact time when will it be finished.

To add something like `DateTime.Now + remainingTimeForRegistration`?

yahiheb replied:
@molnard WDYT?

molnard replied:
In the long term, this message will be removed, it is only for debugging purposes. For me, the current message is enough. @Szpoti if you debug a lot and that is really missing for you, both can be displayed. Is that the case?

molnard reviewed (commented):
Concept ACK - solve the conflict and we can merge.

molnard commented:
It would be also good to display if it is a blame round or not - in a separate PR.

molnard reviewed (approved):
🥇

yahiheb commented:
> Concept ACK - solve the conflict and we can merge.

Fixed the conflicts and removed the hard coded `True` and `False` mentioned here https://github.com/zkSNACKs/WalletWasabi/pull/7628#discussion_r834109660
